src/views/games/Qwixx.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+	<v-container>
+		<v-row class="mx-4 my-4">
+			<h2 class="display-3 font-weight-black underlined">Qwixx</h2>
+		</v-row>
+		<v-row>
+			<input class="mx-4 my-4" v-model="name" placeholder="Put name here" />
+			<v-btn small color="white">Submit</v-btn>
+		</v-row>
+
+		
+		<v-row class="mb-6">
+			<div class="redBtns mr-3 mb-2" v-for="redBtn in redBtns" :key="redBtn.id">
+				<v-btn class="red mark" dark v-on:click="redBtn.marked = !redBtn.marked, redRow++, calculate()">
+					<i v-bind:class="[{ 'white' : redBtn.marked }, 'material-icons']">{{redBtn.name}}</i>
+				</v-btn>
+			</div>
+		</v-row>
+		<!--yellow row-->
+		<v-row class="mb-6">
+			<div class="yellowBtns mr-3 mb-2" v-for="yellowBtn in yellowBtns" :key="yellowBtn.id">
+				<v-btn class="yellow mark" dark v-on:click="yellowBtn.marked = !yellowBtn.marked, yellowRow++">
+					<i v-bind:class="[{ 'white' : yellowBtn.marked }, 'material-icons']">{{yellowBtn.name}}</i>
+				</v-btn>
+			</div>
+		</v-row>
+		<!--green row-->
+		<v-row class="mb-6">
+			<div class="greenBtns mr-3 mb-2" v-for="greenBtn in greenBtns" :key="greenBtn.id">
+				<v-btn class="green mark" dark v-on:click="greenBtn.marked = !greenBtn.marked, greenRow++">
+					<i v-bind:class="[{ 'white' : greenBtn.marked }, 'material-icons']">{{greenBtn.name}}</i>
+				</v-btn>
+			</div>
+		</v-row>
+		<!--blue row-->
+		<v-row class="mb-6">
+			<div class="blueBtns mr-3 mb-2" v-for="blueBtn in blueBtns" :key="blueBtn.id">
+				<v-btn class="blue mark" dark v-on:click="blueBtn.marked = !blueBtn.marked, blueRow++">
+					<i v-bind:class="[{ 'white' : blueBtn.marked }, 'material-icons']">{{blueBtn.name}}</i>
+				</v-btn>
+			</div>
+		</v-row>
+
+		<v-row>
+			<v-col>
+				<p>Penalties</p>
+				<v-btn
+					class="mx-2"
+					color="black"
+					dark
+					@click="subtract(pen.v, pen.e = true)"
+					:disabled="pen.e"
+					v-for="(pen, i) in penalties"
+					:key="i"
+				>{{ pen.v }}</v-btn>
+			</v-col>
+		</v-row>
+
+		<v-row class="mb-6" justify="flex-start" no-gutters>
```

Comment:
I would also change line 59 to <v-row class="mb-6 d-flex justify-start" no-gutters> the justify="flex-start" was throwing an error as well.